MyNotes/Form1.cs

```diff
@@ -1,42 +1,34 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.ComponentModel;
-using System.Data;
-using System.Diagnostics;
-using System.Diagnostics.SymbolStore;
-using System.Drawing;
 using System.IO;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 using System.Windows.Forms;
-using MyNotes;
 
 namespace MyNotes
 {
     public partial class mainForm : Form
     {
         #region Переменные
-        string path = @"Unnamed"; //переменная пути файла
```

Comment:
default visibility of fields are "internal", 

https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/access-modifiers

"internal: The type or member can be accessed by any code in the same assembly, but not from another assembly."

in general you want to keep visibility as tight as possible.

i.e. use "private" everywhere, unless there is a reason to do it differently.